### PR TITLE
Add github actions ci workflow

### PR DESCRIPTION
**Pull request by j616**
Convert to GitHub Actions for CI

Ticket: https://www.pivotaltracker.com/story/show/186493694

### CODE REVIEW

**Review by robwadge — APPROVED** [review]
LGTM - deletion of ci-workflow.yml from .github folder confused me for a moment, but looks like that was just rolling back a recent direct commit to main.  I'm not familiar with tox though, so may want to wait for another review before merging

**Comment by j616** [comment]
@robwadge Yeh. I had a moment where I thought I needed a workflow in main to make this work, then spotted I'd used the wrong directory :upside_down_face:

**Comment by j616** [comment]
@philipnbbc has reviewed https://github.com/bbc/rd-apmm-python-lib-rtpPayload_ttml/pull/7 which has the same tox changes. Shall assume that that review + @robwadge's on this PR means we're happy with all the changes.

**Review by philipnbbc — COMMENTED** [review]
LGTM too. Just wondered why only release Python 3.12 packages, but it's up to you.

File: .github/workflows/release-workflow.yml
Comment (philipnbbc) [reply .github/workflows/release-workflow.yml:21]: Ah yes, you're probably right. The list of supported Python versions shown in pypi is some setup.py metadata rather than something detected from the package.

**Pull request merged**
